views/js/qtiItem/core/Table.js

```diff
@@ -30,5 +31,17 @@ define([
 
     Container.augment(Table);
 
+    Table = Table.extend({
+        // on table creation, we might get a body wrapped in a table element, that we need to get rid of
+        body: function(newBody) {
```

Comment:
sorry, I forgot: why do we need that ?

Reply:
I changed the way the insert callback works for container elements so it would work with tooltips or other containers. As this was only used by table until now, this fix was needed for backward compatibility.